### PR TITLE
Add optimization mods for the modpack

### PR DESCRIPTION
**Pull request by GaryBlackbourne**
One of the main problems we face is with performance. I have added some mods, to ease the load on memory and the CPU.

Mods added:
- Magnesium
- Sulfuric
- Helium
- RoadRunner

### CODE REVIEW

**Comment by GaryBlackbourne** [comment]
The links for the mods:
- [Magnesium](https://www.curseforge.com/minecraft/mc-mods/sodium-reforged)
- [Sulfuric](https://www.curseforge.com/minecraft/mc-mods/phosphor-reforged)
- [Helium](https://www.curseforge.com/minecraft/mc-mods/hydrogen-reforged)
- [RoadRunner](https://www.curseforge.com/minecraft/mc-mods/roadrunner)

**Comment by kiskoza** [comment]
My local setup crashed with these new mods. The error is coming from Helium, which states on its readme that it might be incompatible with Create. Some of the commenters recommends ferritecore instead, please take a look at it and see it it has any overlap / incompatibility with the other three.

Also, when you're ready, don't forget to include the new mods in the modlist.html file as well (bonus point for writing a script to generate that file)

File: manifest.json [hunk 1]
```diff
@@ -267,7 +267,7 @@
     },
     {
       "projectID": 429235,
-      "fileID": 3485636,
+      "fileID": 3435176,
```
Comment (kiskoza) [inline manifest.json]: This one seesm to be the fileID for ferritecore 2.0.7. Before this PR we used to have 2.1.0 and the lastest one is 2.1.1. Please take a look at it again to make sure we have the best option here.

**Review by kiskoza — CHANGES_REQUESTED** [review]
Also, please add the new mods to the modlist.html and write an entry to the changelog

**Review by kiskoza — APPROVED** [review]

**Pull request merged**